storage/innobase/xtrabackup/src/keyring_components.cc

```diff
@@ -160,7 +160,9 @@ bool keyring_init_online(MYSQL *connection) {
 
 bool keyring_init_offline() {
   char fname[FN_REFLEN];
-  if (xtrabackup_stats) {
```

Comment:
do we throw any error if component configration file is not used?

Reply:
@rahulmalik87 If you have not passed a configuration file and there is none in the --target-dir --datadir (in case of status) we will attempt to load the plugin. Then here it follows the current behavior. It won't be able to find the master key when trying to decrypt the table.
If you pass a component config and it has any error in it, then pxb understands that you want to use the component but the config is wrong. In this case, we abort right away.